Core/GDCore/Extensions/Builtin/MathematicalToolsExtension.cpp

```diff
@@ -400,7 +404,8 @@ BuiltinExtensionsImplementer::ImplementsMathematicalToolsExtension(
   extension
       .AddExpression("sin",
                      _("Sine"),
-                     _("Sine of a number"),
+                     _("Sine of an angle (in radian). "
+                       "`ToRad` allows to use degrees."),
```

Comment:
Not clear, rather explain:

```suggestion
                       "If you want to use degrees, use`ToRad`: `sin(ToRad(45))`."),
```